core/rio/api/src/main/java/org/eclipse/rdf4j/rio/helpers/RDFaParserSettings.java

```diff
@@ -26,7 +26,6 @@ public class RDFaParserSettings {
 	 * <p>
 	 * Defaults to {@link RDFaVersion#RDFA_1_0}.
 	 */
-	@Deprecated(since = "4.3.0", forRemoval = true)
```

Comment:
Could you add a comment explaining that these RDFa settings are in use by some users and referencing this issue: https://github.com/eclipse-rdf4j/rdf4j/issues/4779